main.js

```diff
@@ -223,17 +242,24 @@ export class IdDecoder {
    *   with leading non-zero data will error.
    * @param {boolean} [options.multibase=true] - Use multibase encoding to
    *   detect the id format.
-   *
+   * @param {boolean} [options.multihash=false] - Use multihash encoding to
+   *   detect the id format.
+   * @param {number} [options.expectedSize=32] - Optional expected identifier
+   *   size in bytes.
```

Comment:
- Either the docs need to be clear this is only for multihash encoding, or the check should be moved to always run.
- Should this allow falsey values to disable the check?